Pipeline/pipeline.py

```diff
@@ -73,100 +108,99 @@ def __init__(self, df):
         -------
         TYPE
             DESCRIPTION.
 
         """
-        self.X = self.df[self.df.columns[begin:end]]
+        self.X = self.df[self.df.columns[start:end]]
         return self.X
 
-    def set_labels(self, begin, end):
+    def set_labels(self, start, end):
         """
-        
 
         Parameters
         ----------
-        begin : TYPE
-            begin column of dataframe labels
+        start : TYPE
+            Start column of dataframe labels
         end : TYPE
-            end column of dataframe labels
-
+            End column of dataframe labels
 
         Returns
         -------
         TYPE
             DESCRIPTION.
 
         """
-        self.y = self.df[self.df.columns[begin:end]]
+        self.y = self.df[self.df.columns[start:end]]
         return self.y
 
-    def feature_reduction(self, N_top):
+    def feature_reduction(self, n):
         """
-        
 
         Parameters
         ----------
-        N_top : int
+        n : int
             Number of relevant features
 
         Returns
         -------
-        TYPE
-            Minimum number of features that satisfies N_top for each label
+        Minimum number of features that satisfies "n" for each label.
 
         """
         # define the model
         model = DecisionTreeRegressor()
+
         # fit the model
         model.fit(self.X, self.y)
+
         # get importance
         importance = model.feature_importances_
+
         # summarize feature importance
         featureScores = pd.concat(
             [pd.DataFrame(self.X.columns), pd.DataFrame(importance)], axis=1
         )
         featureScores.columns = ["Specs", "Score"]
-        self.best = featureScores.nlargest(N_top, "Score")["Specs"].values
+        self.best = featureScores.nlargest(n, "Score")["Specs"].values
         self.X = self.X[self.best]
+
         return self.X
 
     def data_scaling(self, test_size, scaling=True, scalers=[]):
         """
-        
 
         Parameters
         ----------
         test_size : float
-            percentage of data destined for testing
+            Percentage of data destined for testing.
         scalers : scikit-learn object
             scikit-learn scalers
         scaling : boolean, optional
-            Choose between scaling the data or not. The default is True.
+            Choose between scaling the data or not.
+            The default is True.
 
         Returns
         -------
-        Array
-            X_train - features destined for training.
-        Array
-             X_test -features destined for test.
-        Array
-            y_train - labels destined for training.
-        Array
-             y_test - labels destined for test.
+        X_train : array
+            Features destined for training.
+        X_test : array
+            Features destined for test.
+        y_train : array
+            Labels destined for training.
+        y_test : array
+             Labels destined for test.

```

Comment:
In the validation method we have changed from X to x.
From what I understand tensorflow standard is to use `x_train`, `y_train`... while scikit-learn uses `X_train`, `y_train`...
Since we have already change part of the code from X to x, I think we should also change it here, following the tensorflow standard of `x_train` which is also pep8 compliant.